OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDatesFragment.java

```diff
@@ -50,4 +56,46 @@ public static CourseDatesFragment newInstance(@NonNull Context context,
         fragment.setArguments(makeArguments(context, environment, courseData));
         return fragment;
     }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        EventBus.getDefault().registerSticky(this);
+    }
+
+    @Override
+    public void setUserVisibleHint(boolean isVisibleToUser) {
```

Comment:
`setUserVisibleHint`, `onRevisit`
and `onEvent(NetworkConnectivityChangeEvent event)` functions implementation are repeating in different fragments of View Pager. We should centralized it.